f2/src/utils/yupSchema.js

```diff
@@ -17,10 +22,16 @@ export const yupSchema = () => {
       message: <Trans id="locationInfoForm.Warning" />,
     }),
 
+    phoneExtensionSchema: Yup.string().matches(regexDef().phoneExtensionRegex, {
+      message: <Trans id="contactinfoForm.email.warning" />,
+    }),
+
     dateSchema: {
       DAY: Yup.number().min(1).max(31),
       MONTH: Yup.number().min(1).max(12),
       YEAR: Yup.number().min(1000).max(9999), //Enforce 4 digit entry
     },
   }
 }
+
```

Comment:
Should remove commented out code.

Reply:
removed comment left by others in master, I did not add this comment